components/linear/linear.app.mjs

```diff
@@ -3,6 +3,15 @@ import linearApp from "../linear_app/linear_app.app.mjs";
 export default {
   ...linearApp,
   app: "linear",
+  propDefitions: {
+    ...linearApp.propDefitions,
+    createAsUser: {
+      propDefinition: [
+        linearApp,
+        "createAsUser",
+      ],
+    },
+  },
```

Comment:
Check out [this example in the Pipedream docs](https://pipedream.com/docs/components/api/#prop-definitions-example) to get a better grasp on `propDefinitions`.

The app file should declare the prop's label, description, etc., not import it.
The components that use this app, such as the `create-issue` action, can import it as you did here.

In this case, since you specified the prop's data directly in the action that uses it, there's no need to use propDefinitions at all.